custom/icds_reports/views.py

```diff
@@ -1587,6 +1588,8 @@ def get(self, request, *args, **kwargs):
 @location_safe
 class ICDSImagesAccessorAPI(View):
     @method_decorator(api_auth)
+    @require_permission(Permissions.view_report,
+                        'custom.icds_reports.reports.reports.DashboardReport')
```

Comment:
from trello:

> The IT app will use the API built for the dashboard. I have opened a PR to restrict access to this API to only dashboard users (which include the IT app users). Once that PR is merged, I think the dev side of this will be done.

Do all the IT staff have that permission?

Also how is this role setup? Its not clear to me how reporting permissions work

Reply:
> Do all the IT staff have that permission?

Yup - I was initially planning on using one of the "Issue Tracker" reports, but Marissa pointed out that the dashboard was a better one to use.  That back-and-forth is [here](https://manage.dimagi.com/default.asp?277646).

> Also how is this role setup? Its not clear to me how reporting permissions work

None of it is hardcoded, unfortunately - user roles have a "permissions" object which stores either a `view_reports` boolean indicating whether they can view all reports, or a `view_report_list`, which is a list of the module paths of a whitelist of reports they're allowed to access ([here](https://github.com/dimagi/commcare-hq/blob/3a4343948439e442ec2a2eb51d6f296a3c01379b/corehq/apps/users/models.py#L116-L116)).

This is configured on the "Web Users and Roles" page when editing a user role - uncheck "View all reports", and there's a list of reports to select.